charts/argo-cd/templates/argocd-repo-server/clusterrole.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.repoServer.serviceAccount.create }}
+{{- if .Values.repoServer.clusterAdminAccess.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "argo-cd.repoServer.fullname" . }}
+  labels:
+    {{- include "argo-cd.labels" (dict "context" . "component" .Values.repoServer.name "name" .Values.repoServer.name) | nindent 4 }}
+rules:
```

Comment:
What do you think about reusing what we use for the controller?
https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/values.yaml#L222-L229

Reply:
I hope I understood your idea correctly 😅

Reply:
yep, looks good. will have a deeper look into it as soon as possible